integration_tests/models/source/default/snowplow_mobile_screen_view_context_stg.sql

```diff
@@ -0,0 +1,32 @@
+-- test dataset includes unstruct_event_com_snowplowanalytics_mobile_screen_view_1_0_0 due to a mutual solution covering other adapters
+-- needs to be separated into its own table here
+
+with prep as (
+  select
+    event_id as root_id,
+    collector_tstamp as root_tstamp,
+    substring(unstruct_event_com_snowplowanalytics_mobile_screen_view_1_0_0, position( 'id' in unstruct_event_com_snowplowanalytics_mobile_screen_view_1_0_0)+ 5, 36) as id, -- test dataset uses json format. Extract.
```

Comment:
Because in the mobile model there are more than one key value pairs and each line would be different due to the differing ids. Another issue is the length of one of the values: the session_index is either one or two digits so the logic had to be altered to account for it when I extract that (although this is not the one in the above example)